### PR TITLE
[CI] Reset GPU

### PR DESCRIPTION
**Pull request by alexbatashev**
Reset Intel GPUs before each test run to avoid side effects of GPU driver bugs.

### CODE REVIEW

File: .github/workflows/sycl_linux_build_and_test.yml [hunk 1]
```diff
@@ -200,6 +200,11 @@ jobs:
       image: ${{ matrix.image }}
       options: ${{ matrix.container_options }}
     steps:
+    - name: Reset GPU
+      if: ${{ contains(matrix.config, 'gen9') }}
+      run: |
+        sudo mount -t debugfs none /sys/kernel/debug
+        sudo bash -c 'echo 1 > /sys/kernel/debug/dri/0/i915_wedged'
```
Comment (bader) [inline .github/workflows/sycl_linux_build_and_test.yml:207]: I would prefer a runner to execute in privileged mode only this command.

**Review by bader — APPROVED** [review]

**Comment by bader** [comment]
> Reset Intel GPUs before each test run to avoid side effects of GPU driver bugs.

IMHO, it's not about just GPU driver bugs. It might be a bug in a PR tested on this machine before your run, which left GPU in invalid state.

**Comment by bader** [comment]
BTW, I see that SYCL :: KernelAndProgram/kernel-bundle-merge-options-env.cpp fails on Windows in your pre-commit. Is it a known issue?

**Comment by alexbatashev** [comment]
> > Reset Intel GPUs before each test run to avoid side effects of GPU driver bugs.
> 
> IMHO, it's not about just GPU driver bugs. It might be a bug in a PR tested on this machine before your run, which left GPU in invalid state.

That shouldn't be possible.

**Pull request merged**